docs/changes/0.15.0a1.rst

```diff
@@ -0,0 +1,70 @@
+Changelog for QCoDeS 0.15.0a1
+========================================
+
+The June 2020 alpha release of QCoDeS
+
+Breaking Changes:
+_________________
+
+* ``GroupParameter.group``, ``Group.parameters`` and ``Group.instrument`` are
+  now all read only properties (#2031).
+* The public attributes ``Group.get_cmd`` and `Group.get_cmd`` have
```

Comment:
missing a tick before `Group.get_cmd` (that might be my error :) )

Reply:
Done